### DISCUSSION TITLE
wallet connect transaction is not being sent using wagmi over connected wallets

### DISCUSSION THREAD

**Discussion by can2112** [WalletConnect/walletconnect-monorepo#3449] [Developer Support]
when iam connecting any wallet with walletConnect i am not able to send the transaction over the wallets  @naveenkhasyap

**Comment by glitch-txs** [comment]
Hi could you provide a minimal reproducible example of the issue? It could be on a public repo or StackBlitz or anything similar that we can run by ourselves. Thanks!

**Reply by can2112** [reply]
Hi @glitch-txs  the app which we are using wallet connect using wagmi is a crypto payment gateway where we support onRamp and OffRamp funcationality, for buy it is working, but for sell is there any funcationality where we can send the offRamp transactions to connected wallet,

**Reply by glitch-txs** [reply]
Hi, I'm not familiar with Ramp, what do you mean by OffReamp?

**Reply by can2112** [reply]
offRamp means when  we are selling and onRamp mean we are buying, now when we are selling we have to send transaction to connected wallet and connected wallet should get the transaction and when user approves the transaction  and as soon as we receive the transaction we can send the money to the users bank account. This process is OffRamp

**Reply by can2112** [reply]
@glitch-txs please check

**Reply by glitch-txs** [reply]
hey if you're using Web3Modal with Wagmi this is more on the Wagmi side, maybe try to contact them.
Also if you could share an [MRE](https://github.com/orgs/WalletConnect/discussions/3457) I can take a look :)

**Reply by Naveenkhasyap** [reply]
solved this thanks for the help.
